concent_api/utils/helpers.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+import time
+
+import calendar
+
+from django.utils import timezone
+
+
+def get_current_timestamp() -> int:
+    """ Returns current timestamp as int. """
+    current_timestamp = int(timezone.now().timestamp())
+    assert calendar.timegm(time.gmtime()) - current_timestamp <= 1
+    return current_timestamp
+
+
+def parse_datetime_to_timestamp(date_time: datetime.datetime) -> int:
+    """ Returns timestamp as int from given datetime. """
```

Comment:
Please expand this too:
- Say that we support both naive and timezone-aware `datetime`.
- Naive `datetime` is intepreted as it it were in UTC.
- The resulting timestamp is relative to UNIX epoch and it should work the same on all operating systems.
- The resulting timestamp is also in UTC (and not local time).

Reply:
:+1: